data/mods/Xedra_Evolved/mapgen/mutable/salamander_eruption.json

```
@@ -0,0 +1,725 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "salamander_eruption_ground",
+    "object": {
+      "fill_ter": "t_region_groundcover",
+      "rows": [
+        "                        ",
+        "                        ",
+        "            <<          ",
+        "      ---------         ",
+        "     --,,,,,,,----      ",
+        "    <-,,,,,,,,,,,---    ",
+        "    <-,,,,,,,,,,,,-<    ",
+        "    --,,,,,<,,,,,,-<    ",
+        "   --,,,,,,,,,,,,,--    ",
+        "   <-,,,,,,,,,,,,,,--   ",
+        "   --,,,,,,,,,,,,,,,--  ",
+        "   -,,,,,,,><,,,,,,,,-  ",
+        "   -,,,,,,,><,,,,,,,,-  ",
+        "   -,,,,,,,,,,,,,,,,,-  ",
+        "  <-,,,,,,,,,,,,,,,,,-  ",
+        "  --,,,,,,,,,,,,,,,---  ",
+        "  -,,,,,,,,,,,,,,,,-<   ",
+        " --,,,,,<,,,,,,,<,,-<   ",
+        " <-,,,,,,,,,,,,,,,,---  ",
+        " ---,----,,,,,,,,,,---  ",
+        "   ---<<-,,,,,,---,-<   ",
+        "       <-,------<--     ",
+        "        ---   -         ",
+        "                        "
+      ],
+      "terrain": {
+        " ": [ [ "t_dirt", 2 ], [ "t_region_groundcover", 7 ], [ "t_pit_shallow", 2 ], "t_lava" ],
+        ",": [ "t_obsidian_floor" ],
+        "-": [ "t_obsidian_wall" ],
+        "<": [ "t_ramp_up_low" ],
+        ">": [ "t_ramp_down_high" ]
```

```suggestion
        "<": [ "t_slope_up" ],
        ">": [ "t_slope_down" ]
```
t_ramp_up_low and t_ramp_down_high are for the ramps used for bridges and such where you step up them, t_slope_up and t_slope_down are effectively the same as ladders and are what the mine uses, you can happily use either but the way you've got it set up it needs slope